debian/changelog

```diff
@@ -1,3 +1,68 @@
+snapcraft (2.33) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * tests: reduce the amount of test code in test_meta (#1391)
+  * meta: bash completion support (#1390)
+  * tests: fix issues with python 3.6 (#1398)
+  * cleanbuild: ensure we enter a shell on failures on --debug (#1404)
+  * tests: do not break a systems `bzr whoami` (#1357)
+  * snap: remove completer entry (#1409)
+  * core: minimal cross OS support. (#1413)
+  * core: a clean command should clean. (#1416)
+  * Revert "tests: workaround issue that causes failures to download core (#1386)" (#1423)
+  * nodejs plugin: copy the content of out of tree symlinks (#1418)
+  * New upstream release (LP: #1707060)
+
+  [ Zygmunt Krynicki ]
+  * pluginhandler: don't clobber path on local import failure (#1361)
+
+  [ Paolo Pisati ]
+  * core: use the correct kernel arch for s390 (#1376)
+
+  [ Tim Süberkrüb ]
+  * core: support yaml merge tags (#1384)
+
+  [ roxd ]
+  * blank version should not be allowed in snapcraft.yaml (#1379)
```

Comment:
I'm not sure what section this is about.
Maybe: `validation:`, `schema` or `project loader`.